### PR TITLE
Card

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Card Component.

The page includes:

Header with Girlifycoder Logo, Heading, and Github link
The navigation bar on the left
Sidebar on the right
Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160250879-67edc4c0-fa02-4adb-a636-e50beea78c9a.mp4

### CODE REVIEW

File: cards/styles2-cards.css [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+  font-weight: bold;
+}
+
+.strike-price5 {
+  text-decoration: line-through;
+  color: rgb(117, 117, 117);
+  font-weight: normal;
+}
+.product-discount5 {
+  color: #78068a;
```
Comment (pnchinmay) [inline cards/styles2-cards.css:413]: Use color variables

**Review by pnchinmay — APPROVED** [review]

File: cards/styles2-cards.css [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+.cards-main-heading {
+  color: yellow;
+}
+.cards-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
+  height: 200px;
+  border: 0;
+  transform: scale(1);
```
Comment (neetukumari4858) [inline cards/styles2-cards.css:14]: you should use scale more then one because if we will use 1 then it will not show transform feature.

**Review by neetukumari4858 — APPROVED** [review]
overall looks good

File: cards/styles2-cards.css (hunk 2 above)
Comment (khushijohri2001) [reply cards/styles2-cards.css:14]: actually, it was working and was getting too much of it that's why I have used 1

File: cards/styles2-cards.css (hunk 1 above)
Comment (khushijohri2001) [reply cards/styles2-cards.css:413]: done

**Pull request merged**